### PR TITLE
Feature/stardew

### PR DESCRIPTION
**Pull request by gusteran**
Add a feature to get the invite code for the stardew server. This command grabs the code from the InviteCode.txt file using the provided path in the .env. Modified the readme to denote this addition to the .env file.

### CODE REVIEW

File: commands/stardew.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const fs = require("fs");
+
+module.exports = {
+    description: "Gives the stardew valley invite code!",
+    fun: (msg, tokens, client) => {
+
+        let stardew = process.env.STARDEW_PATH;
+        let appID = "413150"; // The Steam AppID for Stardew
+        const execSync = require('child_process').execSync;
```
Comment (michael-burke4) [inline commands/stardew.js]: move to top

File: readme.md [hunk 2]
```diff
@@ -56,6 +56,8 @@ VarvBot stores api keys in environment variables. In order to set these for your
 
 -RAPID_API_KEY: Varvbot uses rapid api for the love calculator and stock functions. In order to use these apis yourself, you'll need to get an api key from here: https://rapidapi.com/apidojo/api/yahoo-finance1/. If you don't feel like doing this, just delete the stock.js and love.js files from the commands folder and everything should be fine. RapidAPI uses the same key for all of its different apis, so the yahoo finance and love calculator apis work with just the one RAPID_API_KEY. Again, **DONT SHARE THIS WITH ANYONE!!!!**
 
+-STARDEW_PATH: Varvbot can give the invite code of a [Stardew Valley server](https://www.nexusmods.com/stardewvalley/mods/2677) using this directory path to the InviteCode.txt file.
```
Comment (michael-burke4) [inline readme.md:59]: add envexample field

**Review by michael-burke4 — CHANGES_REQUESTED** [review]
small changes! Check out the comments!

File: readme.md (hunk 2 above)
Comment (gusteran) [reply readme.md:59]: @michael-burke4 Fixed

**Review by michael-burke4 — APPROVED** [review]

**Pull request merged**